### ISSUE TITLE
Unable to anchor into seats or other anchor systems.

### ISSUE THREAD

**Issue by Mr-CostlessClaw** [Yellow-Dog-Man/Resonite-Issues#749]
### Describe the bug?

No mater the world, avatar I am in, or anchor type. Neos or Resonite anchor. My avatar will not anchor.

### To Reproduce

This has happened twice. I do not know how to recreate this bug.

### Expected behavior

When I click on an anchor. I am suppos4d to be seated or anchored to any object with an anchor. Via a seat, car, or other anchor systems.

### Resonite Version Number

2023.11.18.1263

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Quest Pro

### Log Files

[DESKTOP-KBDVEQ9 - 2023.11.18.1263 - 2023-11-21 22_50_22.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13433351/DESKTOP-KBDVEQ9.-.2023.11.18.1263.-.2023-11-21.22_50_22.log)

**Comment by epicEaston197** [comment]
Are you sure that you don't have any anti anchor systems on the avatars that you equipped or have you tried to have other people equip the same avatars with the issue to see if they have the same issue as you did that would be really helpful

**Comment by Mr-CostlessClaw** [comment]
No. I tried multiple avatars. One built in game, one that is stripped of systems. I made sure to go through most trouble shooting before posting a ticket.

**Comment by Mr-CostlessClaw** [comment]
Yes. I tried multiple anchors. I do not know what causes it. I've had it happen only one other time. I usually do not anchor, but I have noticed it twice.
